hazelcast/src/main/resources/hazelcast-config-3.9.xsd

```diff
@@ -1362,6 +1362,19 @@
                     </xs:documentation>
                 </xs:annotation>
             </xs:element>
+            <xs:element name="address-locator" type="address-locator" minOccurs="0" maxOccurs="1">
+                <xs:annotation>
```

Comment:
sure, will add it

Reply:
I've added the Spring support as well as the `XmlConfigGenerator`. The changes are in a different PR since I don't have access rights: https://github.com/hazelcast/hazelcast/pull/11548